src/main.ts

```diff
@@ -11,6 +11,7 @@ async function bootstrap() {
         .build();
     const document = SwaggerModule.createDocument(app, options);
     SwaggerModule.setup('api', app, document);
+    app.enableCors();
```

Comment:
Why can't we just pass an option `{ cores: true }` like `const app = await NestFactory.create(AppModule, { cors: true });`
Or because it affects the swagger docs creation?

Reply:
Nope, I just prefer this way, but in fact they both work.